### PR TITLE
[#1075]  NamingEvent add getGroupName and getClusters method

### PR DESCRIPTION
**Pull request by Albert-Hugo**
## What is the purpose of the change

New feature for NamingEvent to support  getGroupName and getClusters method

### CODE REVIEW

File: client/src/main/java/com/alibaba/nacos/client/naming/core/EventDispatcher.java [hunk 1]
```diff
@@ -125,7 +125,7 @@ public void run() {
                     if (!CollectionUtils.isEmpty(listeners)) {
                         for (EventListener listener : listeners) {
                             List<Instance> hosts = Collections.unmodifiableList(serviceInfo.getHosts());
-                            listener.onEvent(new NamingEvent(serviceInfo.getName(), hosts));
+                            listener.onEvent(new NamingEvent(serviceInfo.getName(),serviceInfo.getGroupName(),serviceInfo.getClusters(), hosts));
```
Comment (nkorange) [inline client/src/main/java/com/alibaba/nacos/client/naming/core/EventDispatcher.java]: The code here is not formatted.

Comment (Albert-Hugo) [reply client/src/main/java/com/alibaba/nacos/client/naming/core/EventDispatcher.java]: done

File: api/src/main/java/com/alibaba/nacos/api/naming/listener/NamingEvent.java [hunk 2]
```diff
@@ -28,13 +28,24 @@
 
     private String serviceName;
 
+    private String groupName;
```
Comment (nkorange) [inline api/src/main/java/com/alibaba/nacos/api/naming/listener/NamingEvent.java]: Should consider the default value of groupName and clusterName.

**Comment by nkorange** [comment]
Always associate your commits to issue ID please.

**Comment by Albert-Hugo** [comment]
> Always associate your commits to issue ID please.

Ok.

**Pull request merged**